tests/net/socket/af_packet_ipproto_raw/prj.conf

```diff
@@ -0,0 +1,34 @@
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=n
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_PACKET=y
+CONFIG_POSIX_MAX_FDS=8
+CONFIG_NET_IPV6_DAD=n
+CONFIG_NET_IPV6_MLD=n
+CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=y
+CONFIG_NET_CORE_LOG_LEVEL_DBG=y
+# Network driver config
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+CONFIG_NET_L2_DUMMY=y
+CONFIG_NET_IF_MAX_IPV4_COUNT=4
+
+# Network address config
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
```

Comment:
This doesn't seem to be needed - you only assign the IP address in the main, therefore it unnecessarily delays the initialization (you can observe the test does not start for 30 seconds or so).